core/src/main/java/org/apache/accumulo/core/iterators/SortedKeyValueIterator.java

```diff
@@ -36,11 +36,11 @@
    * Initializes the iterator. Data should not be read from the source in this method.
    *
    * @param source
-   *          <tt>SortedKeyValueIterator</tt> source to read data from.
+   *          <code>SortedKeyValueIterator</code> source to read data from.
```

Comment:
What motivated these `<tt>` to `<code>` changes?

Reply:
Java 11 requires HTML in javadocs to be compliant with HTML5. `<tt>` is not part of HTML5.